types/chrome/index.d.ts

```diff
@@ -5998,6 +5998,63 @@ declare namespace chrome.notifications {
     export function getPermissionLevel(callback: (level: string) => void): void;
 }
 
+////////////////////
+// Offscreen
+////////////////////
+/**
+ * Use the offscreen API to create and manage offscreen documents.
+ * Availability: Since Chrome 109. Manifest v3.
```

Comment:
Can you change this to `@since Chrome 109. Manifest v3.` ?

Reply:
Sorry, forgot about this.
Resolved in 2f0e8940336652e0ddd044a729ea5d171ec88dde